### DISCUSSION TITLE
Login using jetbrains product

### DISCUSSION THREAD

**Discussion by Mehul9833** [community/community#116620] [GitHub Education]
### Select Topic Area

Question

### Body

Hi,

I am not able to access intelij ide from jetbrains.

I have github student plan, but it is saying you dont have any license. 

Please help me out

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Mehul9833 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by cfprocha** [comment]
When you get your InteliJ from Jetbrains, they send you an email, with some finals steps you must follow in order to activate your license, its not just download the software.
One of those steps is to connect another e-mail to your JetBrains account, which may be your personal e-mail, as your student profile is connected to your student e-mail by default. I would strongly recomend you to connect another account.
After that, the first time you open InteliJ you must activate it AND connect to your JetBrains account which have your license. So your first step is to hit button to Login to your JetBrains account and when doing so, you MUST use either your student e-mail or the one you connected as described above.
After that, simply click on Activate and you are all set.
